src/scripts/ui/inline-roll-links.ts

```diff
@@ -349,8 +349,16 @@ export class InlineRollLinks {
         }
 
         const actor = resolveActor(foundryDoc ?? resolveDocument(link));
-        if (actor || pf2Traits) {
+        const item = resolveItem(foundryDoc ?? resolveDocument(link));
```

Comment:
Because you injected itemUuid into the actual anchor's data, foundryDoc should already be the item. So `const item = foundryDoc instanceof ItemPF2e ? foundryDoc : null;` should be enough for your purpose without needing to declare a new helper.

Also, while you're here, please remove `?? resolveDocument(link)` from const actor. I just noticed that foundryDoc is already resolveDocument(link).